horovod/tensorflow/mpi_ops.py

```diff
@@ -76,7 +77,10 @@ def _allreduce(tensor, name=None):
     """
     if name is None and not _executing_eagerly():
         name = 'HorovodAllreduce_%s' % _normalize_name(tensor.name)
-    return MPI_LIB.horovod_allreduce(tensor, name=name)
+    if (group_id is not None):
```

Comment:
Nit: remove parens

Reply:
Removed parens and updated with new default value. I left the if/else statment here to ensure the default `group_id` value is set properly in the C++ backend (as defined by `NULL_GROUP_ID` in `message.h`)